### PR TITLE
Expanded action group documentation

### PR DESCRIPTION
**Pull request by AlchemistCH**
Docs for #1626, and serious expansion of action group docs.

Action group control in kOS definitely needs better explanation, especially with the custom groups kOS creates (I didn't know about them for quite some time, just because they weren't properly explained).

So here's what I propose with this documentation

### CODE REVIEW

**Comment by AlchemistCH** [comment]
The only thing I'd like to add to this is making links in the table in bindings.rst to the corresponding places in commands/flight/systems.rst, but I'm not sure how to do it properly with this kind of table

**Comment by hvacengi** [comment]
If you insert placeholders, I can make sure that the links get added when merged.

**Comment by AlchemistCH** [comment]
Looks good now, at least on the preview. Ah, this table markup gets messy when you put something like this in.

**Comment by AlchemistCH** [comment]
Added NAVMODE documentation #1635

File: doc/source/bindings.rst [hunk 1]
```diff
@@ -242,37 +245,56 @@ These are variables that behave like boolean flags. They can be True or
...
+when the value is set to its current value (if the parts are in different states). 
+Setting the values to these fields will trigger all the corresponding parts that have different state than the one being set. 
+
+
+===================================================================  ==========   =========   ========= ===============
+Variable Name                                                         Can Read     Can Set     Source    Description
+===================================================================  ==========   =========   ========= ===============
+`SAS <commands/flight/systems.html#global:SAS>`__                     yes          yes          stock     (Same as "SAS" indicator on the navball.)
+`RCS <commands/flight/systems.html#global:RCS>`__                     yes          yes          stock     (Same as "RCS" indicator on the navball.)
+`GEAR <commands/flight/systems.html#global:GEAR>`__                   yes          yes          stock     Is the GEAR enabled right now? (Note, since it may be true with all or most gear retracted, you may want to set it off and back on to guarantee everything is deployed).
```
Comment (hvacengi) [inline doc/source/bindings.rst]: I think I'm going to shorten these description entries before merging.  They're fairly long for the table, and get clipped do to width limits anyways.  The detailed information is available at the linked location anyways.

Comment (AlchemistCH) [inline doc/source/bindings.rst]: Yeah, with links to proper documentation on each group such long descriptions are getting redundant

**Pull request merged**